tests/foreman/api/test_contentviewversion.py

```diff
@@ -129,3 +133,67 @@ def test_delete_version_negative(self):
             content_view.version[0].delete()
         # Make sure that content view version is still present
         self.assertEqual(len(content_view.read().version), 1)
+
+    def test_incremental_update_puppet(self):
+        """@Test: Incrementally update a CVV with a puppet module.
+
+        @Assert: The the incremental update succeeds with no errors, and the
+        content view is given an additional version.
+
+        @Feature: ContentViewVersion
+
+        """
+        # Create a content view and add a yum repository to it. Publish the CV.
+        product = entities.Product().create()
+        yum_repo = entities.Repository(
+            content_type='yum',
+            product=product,
+        ).create()
+        content_view = entities.ContentView(
+            organization=product.organization,
+            repository=[yum_repo],
+        ).create()
+        content_view.publish()
+        content_view = content_view.read()
+
+        # Create a puppet repository and upload a puppet module into it.
+        puppet_repo = entities.Repository(
+            content_type='puppet',
+            product=product,
+        ).create()
+        with open(get_data_file(PUPPET_MODULE_NTP_PUPPETLABS), 'rb') as handle:
+            puppet_repo.upload_content(files={'content': handle})
+        puppet_module = entities.PuppetModule(
+            id=content_view.available_puppet_modules()['results'][0]['id']
+        )
+
+        # Incrementally update the CVV with the puppet module.
+        payload = {
+            'content_view_version_environments': [{
+                'content_view_version_id': content_view.version[0].id,
+                'environment_ids': [
+                    environment.id
+                    for environment
+                    in content_view.version[0].read().environment
+                ],
+            }],
+            'add_content': {'puppet_module_ids': [puppet_module.id]},
+            'update_systems': {'included': {}},  # marked as required in docs
```

Comment:
this is perhaps a doc bug, it works fine in my tests without 'update_systems' attribute.